.github/workflows/pipelines.yml

```
@@ -82,7 +82,7 @@ jobs:
   codeql:
     name: call-codeql
     needs: [build,test]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)